model/transaction/event_test.go

```diff
@@ -80,20 +79,34 @@ func TestTransactionEventDecode(t *testing.T) {
 			input: map[string]interface{}{
 				"id": id, "type": trType, "name": name, "result": result,
 				"duration": duration, "timestamp": timestampEpoch,
-				"context": origContext, "marks": marks, "sampled": sampled,
+				"context": map[string]interface{}{"a": "b", "user": map[string]interface{}{
+					"username": name, "email": email, "ip": userIp, "id": userId}, "tags": map[string]interface{}{"foo": "bar"},
+					"page": map[string]interface{}{"url": url, "referer": referer}},
+				"marks": marks, "sampled": sampled,
 				"parent_id": parentId, "trace_id": traceId,
 				"spans": []interface{}{
 					map[string]interface{}{
 						"name": "span", "type": "db", "start": 1.2, "duration": 2.3,
 					}},
 				"span_count": map[string]interface{}{"dropped": 12.0, "started": 6.0}},
-			e: &Event{Id: id, Type: trType, Name: &name, Result: &result,
-				ParentId: &parentId, TraceId: traceId,
-				Duration: duration, Timestamp: timestampParsed,
-				Context: context, Marks: marks, Sampled: &sampled,
+			e: &Event{
+				Id:        id,
+				Type:      trType,
+				Name:      &name,
+				Result:    &result,
+				ParentId:  &parentId,
+				TraceId:   traceId,
+				Duration:  duration,
+				Timestamp: timestampParsed,
+				Marks:     marks,
+				Sampled:   &sampled,
 				SpanCount: SpanCount{Dropped: &dropped, Started: &started},
 				User:      &metadata.User{Id: &userId, Name: &name, IP: &userIp, Email: &email},
 				Labels:    common.MapStr{"foo": "bar"},
+				Page:      &model.Page{Url: &url, Referer: &referer},
```

Comment:
i don't understand why page and context.page are both set in this test... as a matter of fact, it confuses me a bit that any  `Event` can have fields defined twice: as attribute, and inside `context` (this is ofc not introduced in this pr)